content/clients/index.md

```diff
@@ -29,14 +29,16 @@ written in Go with vi-like interface.
 
 ## Web Clients
 
-[netjukebox](http://www.netjukebox.nl/) the flexible media share - netjukebox is a web-based media jukebox for MPD, VideoLAN and Winamp/httpQ. 
```

Comment:
Why does your commit modify this line?

Reply:
because there's a trailing whitespace at the end of the line and my editor cleans it up. It's not deleting the line, just trimming it. I can re-push with the whitespace present if you need it.